docs/content/en/docs-dev/user-guide/event-watcher.md

```diff
@@ -165,6 +165,38 @@ pipectl event register \
 
 Note that it is considered a match only when labels are an exact match.
 
+### [optional] Using contexts
+
+You can also attach additional metadata to the event.
+This information can be added as a trailer to the git commit when Event Watcher using the GIT_UPDATE handler.
+This can be useful when attaching information from the source code repository to the manifest repository.
+
+For example, you can attach the source code commit link to the manifest repository.
+
+```bash
+pipectl event register \
+    --address=CONTROL_PLANE_API_ADDRESS \
+    --api-key=API_KEY \
+    --name=sample \
+    --data=gcr.io/pipecd/helloworld:v0.48.0 \
+    --contexts Source-Commit-Hash=xxxxxxx, Source-Commit-URL=https://github.com/pipe-cd/pipecd/commit/xxxxxxx
```

Comment:
We have to surround the argument with quotes, or when we don't surround it we cannot put a space in the argument.
```suggestion
    --contexts Source-Commit-Hash=xxxxxxx,Source-Commit-URL=https://github.com/pipe-cd/pipecd/commit/xxxxxxx
```

Reply:
Thank you, fixed it at [4a2334a](https://github.com/pipe-cd/pipecd/pull/5299/commits/4a2334a3c96d31c9bcfcbfae02eb6873dd923cdc)